src/core/zowe/core_for_zowe_sdk/sdk_api.py

```diff
@@ -33,12 +33,15 @@ class SdkApi:
         Name of the logger (same as the filename by default)
     """
 
-    def __init__(self, profile: dict, default_url: str, logger_name: str = __name__):
+    def __init__(self, profile: dict, default_url: str, logger_name: str = __name__, log = False):
```

Comment:
I think we want logging enabled by default, so the default value of the `log` argument should be True.